pkg/contracts/constants.go

```diff
@@ -107,3 +107,21 @@ func (d DerivedComponent) Validate() bool {
 	}
 	return false
 }
+
+type LayerType string
+
+const (
+	ApplicationLayer LayerType = "app"
```

Comment:
Remove "Layer" from the different entries as it's a redundant term since each entry will be fully qualified by its type in usage.

Example
`x := LayerType.Application`

Reply:
I agree with removing the redundant "Layer" term for clarity. Similarly, do you think we'll need to update the `AnnotationType` constants to eliminate the "Annotation" prefix for consistency and brevity?